lib/inv_ca_cg.cpp

```diff
@@ -36,26 +36,38 @@ namespace quda {
                          // param.precision == param.precision_sloppy &&
                          // param.use_init_guess == QUDA_USE_INIT_GUESS_NO);
 
+      // printf("5\n");
```

Comment:
Looks like something went awry with branches and `clang-format`...

Reply:
It was not clang-format's fault. I did not clean up the debug `printf`'s before `clang-format` and did a `git commit -a` afterwards ...